summary/summary.py

```diff
@@ -2,7 +2,7 @@
 from pyln.client import Plugin, Millisatoshi
 from packaging import version
 from collections import namedtuple
-from datetime import datetime
+from summary_avail import *
```

Comment:
If you're concerned that the path might not be unique you can explicitly import a local module by using the following:

```python
from .availability import *
```

This saves you from having to prefix everything. Alternatively you could also have the executable in the top-level and create a `summary` package containing all the sub-modules.

Reply:
Wont do, since plugin API or autoreload plugin gets me this:

```
    from .availability import *
ImportError: attempted relative import with no known parent package
```